docs/hub/_toctree.yml

```
@@ -123,6 +123,8 @@
     title: Static HTML Spaces
   - local: spaces-sdks-python
     title: Custom Python Spaces
+  - local: spaces-embedding
+    title: Embedding your Space
```

```suggestion
  - local: spaces-embed
    title: Embed your Space
```
i would not call it "embedding" as ML people see this term and think of something else (a n-vector)

---

Simply replace it with Sharing then maybe ?

---

embed is fine IMO

---

it's specifically "embedding" that might be confusing